src/sdk/namespace_fs.js

```diff
@@ -1419,16 +1427,14 @@ class NamespaceFS {
     }
 
     // move to dest GPFS (wt) / POSIX (w / undefined) - non part upload
-    async _move_to_dest(fs_context, source_path, dest_path, target_file, open_mode, key, is_dir_content) {
-        dbg.log2('_move_to_dest', fs_context, source_path, dest_path, target_file, open_mode, key, is_dir_content);
+    async _move_to_dest(fs_context, source_path, dest_path, target_file, open_mode, key, is_old_dir_flow) {
```

Comment:
I still think is_old_dir_flow is not an understood name, please change it to explicit name and add comments on what it is